data/src/message/source.rs

```diff
@@ -70,11 +64,13 @@ pub mod server {
         Part,
         Quit,
         ReplyTopic,
+        StatusSuccess,
+        StatusError,
```

Comment:
We shouldn't need these new new message kinds, we already cover this by tagging the messages as `Source::Internal`

Reply:
Addressed in the latest refactor push!